src/messaging/ExchangeMgr.cpp

```diff
@@ -70,7 +71,7 @@ CHIP_ERROR ExchangeManager::Init(SecureSessionMgrBase * sessionMgr)
 
     mNextExchangeId = GetRandU16();
 
-    memset(ContextPool, 0, sizeof(ContextPool));
+    memset((void *) ContextPool, 0, sizeof(ContextPool));
```

Comment:
What's this for?

Reply:
ContextPool is a non-trival type, compiler on Pi4 is not happy to see memset on it.

Reply:
We shouldn't try to trick the compiler when it identifies a problem. That said, I can't reproduce this warning.